spark/src/main/java/org/apache/zeppelin/spark/DepInterpreter.java

```diff
@@ -183,7 +183,9 @@ public InterpreterResult interpret(String st, InterpreterContext context) {
 
     if (sparkInterpreter != null && sparkInterpreter.isSparkContextInitialized()) {
       return new InterpreterResult(Code.ERROR,
-          "Must be used before SparkInterpreter (%spark) initialized");
+          "Must be used before SparkInterpreter (%spark) initialized\n" +
+          "Hint: put this cell before any spark/scala code or/and " +
+          "restart Zeppelin" );
```

Comment:
@xhudik I think we should use the term 'paragraph' instead of cell, we don't have anything named cell in Zeppelin. Otherwise this is a very useful message. Thanks